docs/source/api/v5/cdns.rst

```diff
@@ -135,6 +138,7 @@ Response Structure
 :domainName:    The top-level domain (TLD) assigned to the newly created CDN
 :id:            An integral, unique identifier for the newly created CDN
 :name:          The newly created CDN's name
+:ttlOverride:	A :abbr:`TTL (Time To Live)` value, in seconds, that, if set, overrides all set set TTL values on :term:`Delivery Services` in this :term:`CDN`
```

Comment:
Same

Reply:
Not repeating `set` in d81596bedf

Reply:
Changed `CDN` to an abbreviation in 53c02be419